test/common/test_helper.exs

```diff
@@ -1,20 +1,6 @@
 ExUnit.start(capture_log: true)
 
-defmodule TimeHelper do
-  def wait_until(fun), do: wait_until(500, fun)
-
-  def wait_until(0, fun), do: fun.()
-
-  def wait_until(timeout, fun) do
-    try do
-      fun.()
-    rescue
-      ExUnit.AssertionError ->
-        :timer.sleep(10)
-        wait_until(max(0, timeout - 10), fun)
-    end
-  end
-end
+HTTPoison.start()
 
 defmodule TestHelper do
```

Comment:
This still seems to be duplicated (saw that in same other file).

Reply:
Removed from other place now